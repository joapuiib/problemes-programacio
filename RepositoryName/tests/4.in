16 1 joan.puigcerver.7e2@itb.cat
