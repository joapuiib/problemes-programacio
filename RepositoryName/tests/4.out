joanpuigcerver7e2-dam-m16-uf1
